kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/consistencycheck/ConsistencyCheckJob.java

```diff
@@ -74,12 +70,9 @@ public void stop() {
             log.info("stop consistency check job, jobId is null, ignore");
             return;
         }
-        for (PipelineTasksRunner each : getTasksRunnerMap().values()) {
+        for (PipelineTasksRunner each : getTaskRunners()) {
             each.stop();
         }
-        getTasksRunnerMap().clear();
-        String jobBarrierDisablePath = PipelineMetaDataNode.getJobBarrierDisablePath(getJobId());
-        pipelineDistributedBarrier.persistEphemeralChildrenNode(jobBarrierDisablePath, 0);
-        PipelineJobProgressPersistService.removeJobProgressPersistContext(getJobId());
```

Comment:
`removeJobProgressPersistContext` is removed here, but not in migraiton job.

1, Is `removeJobProgressPersistContext` not needed for consistency job?

2, Could `removeJobProgressPersistContext` be added in AbstractPipelineJob?

Reply:
Ok, I will extract `removeJobProgressPersistContext` method to parent class